### PR TITLE
feat: Add basic issue forms

### PR DESCRIPTION
**Pull request by Panquesito7**
### Describe your change:

- Add basic issue forms.

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by krishguptadev — APPROVED** [review]

**Comment by WISSAL-MN** [comment]
https://github.com/TheAlgorithms/Python/pull/5822#issue-1052395608

**Comment by Panquesito7** [comment]
@AnupKumarPanwar.

**Comment by Panquesito7** [comment]
@AnupKumarPanwar.

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Feature request
+description: Suggest features, propose improvements, discuss new ideas.
+title: "[FEATURE]"
```
Comment (dhruvmanila) [inline .github/ISSUE_TEMPLATE/feature_request.yml:3]: Same suggestion, not a big fan of prefixed title. We have labels for that.

**Review by dhruvmanila — CHANGES_REQUESTED** [review]
Some initial thoughts. Also, I don't see any point of having the `other` issue form.

**Comment by Panquesito7** [comment]
> I don't see any point of having the `other` issue form

That's so people cannot make blank issues without any content. They have to provide some content if none of the provided issue templates fits their problem. Without this, people won't be able to make any other miscellaneous issues if we disable blank issues. Hope you understand. 🙂

**Comment by dhruvmanila** [comment]
closing in favor of #6533

**Pull request closed without merging**